infra/us-east-1/prod/vpc/terragrunt.hcl

```diff
@@ -9,6 +9,9 @@ locals {
   project     = local.environment_vars.locals.project
   tribe       = local.account_vars.locals.tribe
   name        = "${local.project}-${local.env}-${local.region}"
+  # New VPC configuration
+  new_vpc_cidr = local.environment_vars.locals.new_vpc_cidr
+  new_vpc_public_subnets = local.environment_vars.locals.new_vpc_public_subnets
```

Comment:
I believe that these lines are not needed as they are from definition of another VPC.